src/test/tables/int_float.tbl

```diff
@@ -1,5 +1,5 @@
 a|b
-int|float
+int|double
```

Comment:
Should change file name upon changing column type. Should also change the input for any test cases in which floats should be compared, i.e. input `400.1f` instead of `400.1`.